scripts/globus/setup_collection_directory.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+set -euf -o pipefail
+
+SCRIPT=$(realpath "$0")
+SOURCE=$(dirname "$SCRIPT")
+PROJECT_ROOT=$(realpath "${SOURCE}"/../../)
+
+list_compose_directories() {
+  echo "Available compose directories:"
+  if composes=$(ls -l "${PROJECT_ROOT}/compose/" | grep "^d") ; then
+    echo "$composes"
+  else
+    echo "No compose directories found."
+  fi
+}
+
+if [ "$#" -ne 1 ]; then
+  echo "Usage: $(basename "$0") <compose_directory>"
+  list_compose_directories
+  exit 1
+fi
+
+COMPOSE_DIR=$1
```

Comment:
Is there a reason you did not include "" around the "$1"?

Reply:
No, a miss